### PR TITLE
Update defender-endpoint-plan-1-2.md

### LINKED ISSUE #11766: Windows 11 client missing
[Enter feedback here]

When is defining OS supported should be say windows 10/11 ?
---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: ec57328d-4b17-abe6-001e-c5ecb2f00746
* Version Independent ID: be98599a-b690-a3e6-c610-8da71fd80d76
* Content: [Compare Microsoft endpoint security plans](https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/defender-endpoint-plan-1-2?view=o365-worldwide)
* Content Source: [microsoft-365/security/defender-endpoint/defender-endpoint-plan-1-2.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender-endpoint/defender-endpoint-plan-1-2.md)
* Service: **microsoft-365-security**
* Sub-service: **mde**
* GitHub Login: @denisebmsft
* Microsoft Alias: **deniseb**

### PR DESCRIPTION
**Pull request by msbemba**
Updated supported OS versions

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/11766

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/defender-endpoint-plan-1-2.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ The following table summarizes what's included in Microsoft endpoint security pl
...
+| [Defender for Endpoint Plan 1](defender-endpoint-plan-1.md)  | - [Next-generation protection](defender-endpoint-plan-1.md#next-generation-protection) (includes antimalware and antivirus)<br/>- [Attack surface reduction](defender-endpoint-plan-1.md#attack-surface-reduction)<br/>- [Manual response actions](defender-endpoint-plan-1.md#manual-response-actions)<br/>- [Centralized management](defender-endpoint-plan-1.md#centralized-management)<br/>- [Security reports](defender-endpoint-plan-1.md#reporting)<br/>- [APIs](defender-endpoint-plan-1.md#apis)<br/>- [Support for Windows 10, Windows 10, iOS, Android OS, and macOS devices](defender-endpoint-plan-1.md#cross-platform-support)|
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/defender-endpoint-plan-1-2.md]:
```suggestion
| [Defender for Endpoint Plan 1](defender-endpoint-plan-1.md)  | - [Next-generation protection](defender-endpoint-plan-1.md#next-generation-protection) (includes antimalware and antivirus)<br/>- [Attack surface reduction](defender-endpoint-plan-1.md#attack-surface-reduction)<br/>- [Manual response actions](defender-endpoint-plan-1.md#manual-response-actions)<br/>- [Centralized management](defender-endpoint-plan-1.md#centralized-management)<br/>- [Security reports](defender-endpoint-plan-1.md#reporting)<br/>- [APIs](defender-endpoint-plan-1.md#apis)<br/>- [Support for Windows 10, Windows 11, iOS, Android OS, and macOS devices](defender-endpoint-plan-1.md#cross-platform-support)|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@denisebmsft Please merge this PR. Thanks

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba

**Pull request merged**